api/SVGElement.json

```diff
@@ -675,40 +675,40 @@
           "spec_url": "https://drafts.csswg.org/cssom-view/#eventdef-window-resize",
```

Comment:
I think we should remove this entry, the actual event is the global "resize" event represented by api.Window.resize_event. I can't find any "resize" event being dispatched in Chromium's SVG code, at least, and nothing in the spec.